tests/problem3.test.js

```diff
@@ -56,3 +56,11 @@ describe('problem3', () => {
         });
     });
 });
+
+
+function first(functions) {
+
+    return functions[0]
```

Comment:
Zrobiliśmy udpate testów do tego zadania. To niestety nie jest dobre rozwiązanie. ;)